english/knife.md

```diff
@@ -0,0 +1,6 @@
+You stab this miterious creature over and over 
```

Comment:
Thank you for the feedback

On Sun, Jan 31, 2016 at 1:46 PM, Casey Collaborator <
notifications@github.com> wrote:

> In english/knife.md
> <https://github.com/udacity/create-your-own-adventure/pull/3189#discussion_r51359276>
> :
>
> > @@ -0,0 +1,6 @@
> > +You stab this miterious creature over and over
>
> It looks like this file has been added to a directory that already
> contained a "page" of the story. Please create a new directory for each new
> page to help keep the repository organized. See the README of this
> repository for more information.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/3189/files#r51359276>
> .
>